de-lib/src/main/java/org/iplantc/de/notifications/client/presenter/NotificationPresenterImpl.java

```diff
@@ -160,6 +146,7 @@ private void addEventHandlers() {
     public void onNotificationGridRefresh(NotificationGridRefreshEvent event) {
         if (listStore.size() > 0) {
             toolbar.setDeleteAllButtonEnabled(true);
+            toolbar.setMarkAsSeenButtonEnabled(true);
         } else {
             toolbar.setDeleteAllButtonEnabled(false);
```

Comment:
Good catch but I think line ` toolbar.setMarkAsSeenButtonEnabled(true);` should be removed since we don't want 'mark as seen' button enabled on Grid refresh event. We want them enabled only when an unread notification is selected. Please see line 201. I will commit a change to remove this code from `onNotificationGridRefresh()` method and re-open this PR against master. Thanks for the review.